booster_bdd/features/steps/launchBooster.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from behave import *
+from features.src.importBooster import *
+from features.src.support import *
+
+@given(u'I have a space created from which I can launch a new booster')
+def step_impl(context):
+    global spaceID
+    spaceID = helpers.getSpaceID()
+    assert spaceID is not None
```

Comment:
Please do not use the plain `assert` statements, use `pyshould` library instead.
See https://github.com/fabric8io/fabric8-test/pull/886, thanks.